Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
 ARG RUSTC_VERSION=1.67.1
 RUN apk update \
     && apk upgrade \
-    && apk add build-base binutils-gold openssl3-dev protoc protobuf-dev curl git \
+    && apk add build-base binutils-gold openssl3-dev protoc protobuf-dev curl git linux-headers clang \
```

Comment:
there may be issues with the new ubi8 based images https://github.com/Kuadrant/limitador/pull/168

Reply:
Totally will be...